dev/fattest.simplicity/autoFVT-defaults/src/ant/launch.xml

```diff
@@ -691,6 +691,25 @@
 			<exclude name="${javacores.server.exclude}" />
 		</fileset>
 
+		<!-- Move any test javacores into the javacores directory that were piped to /var, there are two places to check -->
+		<fileset id="javacores.test" dir="/var/crash/">
```

Comment:
You will need to make different refids and then add them to the move down below approx line 702